docs/guide/Titanium_SDK/Titanium_SDK_Guide/Hyperloop/Hyperloop_Guides/Hyperloop_Requirements.md

```diff
@@ -9,13 +9,11 @@ weight: '10'
 
 You’ll need to have the following installed on your machine as the recommended requirements to use Hyperloop 5.0.0+:
 
-* Appcelerator Studio: 5.0.0+
-
 * Titanium SDK: 9.0.0+
 
 * Titanium CLI: 7.0.0+
 
-You can install [Appcelerator Studio](https://platform.axway.com/#/product/studio) and [Titanium CLI](https://platform.axway.com/#/product/cli) to get the latest version of all these components.
+You can install [Titanium CLI](https://platform.axway.com/#/product/cli) to get the latest version of all these components.
```

Comment:
If we have a page on titaniumsdk.com link that or https://github.com/tidev/titanium

Reply:
there it is: /guide/Titanium_SDK/Titanium_SDK_Guide/Titanium_Command-Line_Interface_Reference/